src/platform/plugins/shared/dashboard/server/content_management/v3/cm_services.ts

```diff
@@ -302,6 +307,29 @@ export const panelSchema = schema.object({
   ),
 });
 
+export const sectionSchema = schema.arrayOf(
+  schema.object({
+    id: schema.maybe(
+      schema.string({
+        meta: { description: 'The unique ID of the section.' },
+      })
+    ),
+    order: schema.number({
```

Comment:
Maybe not something that needs to change, but I wonder if we really need to persist the `order` property since the persisted array can be considered ordered. Maybe the UI state needs the `order` property, but maybe it could be dropped in the `convertSectionMapToSectionArray` if the returned array is sorted?

Reply:
🤔 That's an interesting idea!! I have a follow-up question to that. 

For the Dashboard API, are you able to make a partial update request? So for example, if I have a dashboard with 2 sections and I want to add a third, do I have to redefine the first two sections in my update request, or is the request spread on top of the existing two?

If it's the first option, then I think this would work - but if it's the latter, I think this would prevent users from updating the order of their sections programmatically?

Reply:
Let's leave the `order` property alone. I'm thinking way outside the scope of this PR. 😅  We can always make the `order` property optional later.